src/profiles/y_profile_admin_classes.prog.abap

```diff
@@ -559,6 +565,9 @@ CLASS lcl_util IMPLEMENTATION.
         checks_tree->set_field_header_text( fieldname   = 'APPLY_ON_TESTCODE'
                                             header_text = 'Apply on Testcode'(034) ).
 
+        checks_tree->set_field_header_text( fieldname   = 'IGNORE_PSEUDO_COMMENTS'
+                                            header_text = 'Allow Exemptions' ). "Cause of usability, the text is switched on the UX side.
```

Comment:
The comment is not in the right position.

Reply:
just edit it